Zombie.java

```diff
@@ -1,6 +1,17 @@
 public class Zombie extends SmoothMover {
+	
+	public int strength;
+	public int resistance;
+	public int speed;
+	
 	public Zombie() {
 		//TODO: image is temporarily
 		setImage("./ppl2.png");
 	}
```

Comment:
ToDo:
- add constructor call to default constructor with default values and move `setImage()` call to other constructor